cli/popper/commands/cmd_run.py

```diff
@@ -34,10 +34,25 @@
     required=False,
     default=None
 )
+@click.option(
+    '--recursive',
+    help='Run any main.workflow found recursively from current path. ',
```

Comment:
This mentions `main.workflow` files but the search below is for any `.workflow` file. It's better if we make it for any `.workflow` file, so we need to update the `help` string so it reads any `Any .workflow file`.

Reply:
changed it to `add .workflow`